haskell-names.cabal

```diff
@@ -246,6 +246,9 @@ Library
     , bytestring >= 0.10.4.0 && < 0.11
     , data-lens-light >= 0.1.2.1 && < 0.2
     , traverse-with-class >= 1.0.0.0 && < 1.1
+    , template-haskell >= 2.9 && < 2.14
+    , hint >= 0.7 && < 0.9
+    , th-lift, th-lift-instances, th-orphans
```

Comment:
I would like to avoid adding more dependencies to `library` target. Could you try to implement this feature as a separate executable? Though this really can be useful. This feature is quite big though... It might be convenient to use it directly from library. But it looks more like an idea for different project rather that patch for `haskell-names`.